icons/database-zap.svg

```diff
@@ -0,0 +1,16 @@
+<svg
+  xmlns="http://www.w3.org/2000/svg"
+  width="24"
+  height="24"
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <ellipse cx="12" cy="5" rx="9" ry="3" />
+  <path d="M3 12c0 1.7 4 3 9 3 .7 0 1.4 0 2-.1" />
+  <path d="M3 5v14c0 1.7 4 3 9 3 1.2 0 2.4-.1 3.4-.2" />
+  <path d="m21 10-3 5h4l-3 5" />
```

Comment:
Some optimization + I think the zap could be more aligned horizontally with the refresh icon:
```suggestion
  <ellipse cx="12" cy="5" rx="9" ry="3" />
  <path d="M3 5V19A9 3 0 0 0 15 21.84" />
  <path d="M21 5V8" />
  <path d="M21 12L18 17H22L19 22" />
  <path d="M3 12A9 3 0 0 0 14.59 14.87" />
```

![image](https://github.com/lucide-icons/lucide/assets/17746067/b590d75d-68d9-4871-8f2c-851ea15a6242)

Reply:
@karsa-mistmere I was going more for lining up the middle horizontal lines visually…